### PR TITLE
Add ImportConsult command

### LINKED ISSUE #180: Add ImportConsult command
Currently, we are only able to import students' data from a .csv file.

Let's add this functionality for consultations as well.

### PR DESCRIPTION
**Pull request by yhanyi**
Fixes #180 

Adds ability to import consultation data from a .csv file, similar to that of students.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+            }
+
+            String line;
+            boolean hasValidEntries = false;
+            while ((line = reader.readLine()) != null) {
+                if (line.trim().isEmpty()) {
+                    continue;
+                }
+                hasValidEntries = true;
+                try {
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java]: Is it possible to extract this part into its own private method? The execute method seems a bit long.

File: src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            throw new IllegalArgumentException("Incomplete consultation entry");
+        }
+
+        try {
+            String date = parts[0].trim();
+            String time = parts[1].trim();
+            String studentsStr = unescapeSpecialCharacters(parts[2]);
+
+            List<Student> students = new ArrayList<>();
+            if (!studentsStr.isEmpty()) {
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java]: This potentially falls into arrowhead code territory. Maybe consider extracting the code into another method?

**Review by notnotmax — COMMENTED** [review]
The functionality *seems* fine to me, because this isn't exactly my expertise. However, I have a few concerns regarding code quality, specifically nested code and weak SLAP - though if it is too awkward to extract the logic into its own method then just let me know.

File: src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+            String header = reader.readLine();
+            if (header == null) {
+                throw new CommandException(MESSAGE_EMPTY_FILE);
+            }
+
+            if (!header.equalsIgnoreCase("Date,Time,Students")) {
+                throw new CommandException(MESSAGE_INVALID_HEADER);
+            }
+
+            //            String line;
```
Comment (notnotmax) [inline src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java]: Did you perhaps... forget to remove this? 🤔

Comment (yhanyi) [reply src/main/java/seedu/address/logic/commands/consultation/ImportConsultCommand.java]: Perchance 🤌

**Comment by notnotmax** [comment]
hi your PR conflicts with your other PR lol

**Pull request merged**